src/biogeochem/CNCStateUpdate3Mod.F90

```diff
@@ -177,8 +201,26 @@ subroutine CStateUpdate3( num_soilc, filter_soilc, num_soilp, filter_soilp, &
               cf_veg%m_gresp_xfer_to_fire_patch(p) * dt
          cs_veg%gresp_xfer_patch(p) = cs_veg%gresp_xfer_patch(p) -                 &
               cf_veg%m_gresp_xfer_to_litter_fire_patch(p) * dt
+        else
+         cs_veg%gresp_storage_patch(p) = cs_veg%gresp_storage_patch(p) -           &
+              cf_veg%m_gresp_storage_to_fire_patch(p) * dt
+         cs_veg%gresp_storage_patch(p) = cs_veg%gresp_storage_patch(p) -           &
+              cf_veg%m_gresp_storage_to_litter_fire_patch(p) * dt
+         cs_veg%gresp_xfer_patch(p) = cs_veg%gresp_xfer_patch(p) -                 &
+              cf_veg%m_gresp_xfer_to_fire_patch(p) * dt
+         cs_veg%gresp_xfer_patch(p) = cs_veg%gresp_xfer_patch(p) -                 &
+              cf_veg%m_gresp_xfer_to_litter_fire_patch(p) * dt  
+        end if !end use_matrixcn
+!         if (is_end_curr_year())then
+!            write(begp+1104000000,"(I,18E17.9)"),p,cs_veg%leafc_patch(p),cs_veg%leafc_storage_patch(p),cs_veg%leafc_xfer_patch(p),&
+!                                                cs_veg%frootc_patch(p),cs_veg%frootc_storage_patch(p),cs_veg%frootc_xfer_patch(p),&
+!                                       cs_veg%livestemc_patch(p),cs_veg%livestemc_storage_patch(p),cs_veg%livestemc_xfer_patch(p),&
+!                                       cs_veg%deadstemc_patch(p),cs_veg%deadstemc_storage_patch(p),cs_veg%deadstemc_xfer_patch(p),&
+!                                    cs_veg%livecrootc_patch(p),cs_veg%livecrootc_storage_patch(p),cs_veg%livecrootc_xfer_patch(p),&
+!                                    cs_veg%deadcrootc_patch(p),cs_veg%deadcrootc_storage_patch(p),cs_veg%deadcrootc_xfer_patch(p)
+!         end if
```

Comment:
1. This is the only time using `is_end_curr_year()` which is commented out. Please remove it from line 11.
 
2. Please remove all the above commented out lines.

Reply:
done